js/songs/new.html

```diff
@@ -0,0 +1,11 @@
+<div class="container">
+  <div class="jumbotron audio-player center-block col-md-offset-3">
+<form class="" ng-show="editable" action="index.html" method="post">
+  <input type="text" name="artist" value="" paceholder="artist" ng-model="artist">
+  <input type="text" name="artist" value="" paceholder="title" ng-model="title">
```

Comment:
think the `name` should attribute of each input should reflect the values passed in